otter/test/integration/test_utils.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests for the utility functions for convergence black-box testing.
+"""
+from pyrsistent import pmap, pset
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.convergence.model import (
```

Comment:
I don't recall what we were going to use these for.  I think for re-using any JSON marshaling in Otter?  @lvh can you confirm?